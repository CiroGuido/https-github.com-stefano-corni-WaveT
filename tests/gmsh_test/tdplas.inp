drl ! Debye (deb) vs Drude-Lorentz (drl)
1000. 1.000 0.110224 0.00757576 0.0 0.0    ! eps_0,eps_d,eps_A,eps_gm,eps_w0,f_vel
!
gmsh 
