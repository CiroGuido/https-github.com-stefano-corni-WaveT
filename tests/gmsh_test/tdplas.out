Done reading input, took     0.000s
 nts,nts after eliminating replica         758         758
 Tot. area   3.11502085916033     
 Created output file with surface points
 Matrixes S D have been written out, I'll stop now
Done setting S and D, took     2.138s
Done, total elapsed time     2.138s
